src/__stories__/Forms/Button.tsx

```diff
@@ -9,15 +9,23 @@ import { storiesOf } from '@storybook/react';
 import { Button, IButton } from '../../Button';
 import { boxKnobs } from '../Layout/Box';
 
-export const buttonKnobs = (tabName = 'Button'): Partial<IButton> => ({
-  ...omit(boxKnobs(), 'opacity'),
-  disabled: boolean('disabled', false, tabName),
-});
+import { cleanKnobs } from '../_utils';
+
+export const buttonKnobs = (tabName = 'Button'): Partial<IButton> => {
+  return cleanKnobs<IButton>({
+    ...omit(boxKnobs(), 'opacity'),
+    disabled: boolean('disabled', false, tabName),
+  });
+};
 
 storiesOf('Forms:Button', module)
   .addDecorator(withKnobs)
 
-  .add('with defaults', () => <Button {...buttonKnobs()}>Button Text</Button>)
+  .add('aaaisabled', () => <Button>Button Text</Button>)
```

Comment:
```suggestion
```
I believe you didn't mean to commit this.

Reply:
oops